configuration/sitemaps.md

```diff
@@ -451,15 +459,17 @@ Splits a String Item at each separator into multiple rows.
 
 Mappings is an optional parameter for the [Switch](#element-type-switch) and [Selection](#element-type-selection) element types.
 
-Please be aware of the fact, that both Switch and Selection are input element types.
-If you are looking to transform Item data into meaningful outputs, a [Text element](#element-type-text) with it's label parameter may be a better choice.
+Please note that Switch and Selection are input-only element types.
+In other words, they only accept commands.
+For more information, see [Commands vs. Status]({{base}}/configuration/items.md#command-vs-status) in the [Items]({{base}}/configuration/items.md) article.
+
+Mapping syntax:
```

Comment:
I tried to reduce these short paragraphs as much as possible as they do not look good in the final document. That's why there was a bigger paragraph in between the two code blocks here.

Reply:
So I deleted this paragraph per the above.

Reply:
This comment has been addressed.